plugins/misp/plugin.spec.yaml

```diff
@@ -135,18 +144,22 @@ types:
     event:
       title: Event ID
       description: Event ID to append to
+      example: "1"
       type: string
     other:
       title: Other
       description: Other
+      example: "Other"
       type: string
     comment:
       title: Comment
       description: Attribute comment
+      example: "Example attribute comment"
       type: string
     distribution:
       title: Distribution
       description: Distribution type
+      example: "All Communities"
```

Comment:
When there is a enum - "This Community", shouldn't be the example a valid one?

Reply:
Line 168, there is enum like `All Communities`